packages/web/src/components/EventsList/EventsList.tsx

```diff
@@ -78,7 +78,7 @@ const EventCard: React.FC<EventCardProps> = ({ event }) => {
 export const EventsList: React.FC<{ events: Event[] }> = ({ events }) => (
   <UnorderedList variant="card">
     {events.map((event) => (
-      <EventCard {...{ event, key: event.id }} key={`event-${event.id}`} />
+      <EventCard {...{ event }} key={`event-${event.id}`} />
```

Comment:
Good catch; the linter was unhappy with the other approach as it didn't find it for some reason